modules/smiley.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+'''
+smiley.py - Smiley Module
+Copyright 2014 Sujeet Akula (sujeet@freeboson.org)
+Licensed under the Eiffel Forum License 2.
+
+More info:
+ * jenni-misc: https://github.com/freeboson/jenni-misc/
+ * jenni: https://github.com/myano/jenni/
+ * Phenny: http://inamidst.com/phenny/
+'''
+
+import io, random, os
+
+def smiley(jenni, input):
+    '''.smiley -- print a random smiley'''
+
+    with io.open(os.getcwd() + '/modules/smileys.txt', 'r', encoding='utf-16-le') as f:
```

Comment:
@freeboson I think an alternative to this would be to have an additional files dir and move it into place into jenni's hidden directory on first run. Still waiting on an opinion from @myano though